docs/release-notes/5.7.44-49.md

```diff
@@ -0,0 +1,160 @@
+# Percona Server for MySQL 5.7.44-49 (2024-02-XX)
+
+
+This release is part of the 5.7 EOL program. The fixes are available to [Post MySQL 5.7 EOL support customers]. Community members can [build this release from source].
```

Comment:
thanks for improving this one, and sorry have not had enough time to get back earlier or suggest exact changes to make.
but wanted to flag something else, in the meantime.  i recommend we link to our own page on how to compile. If for any reason we need to link to an external page, we should still at least share which repository and branch to use: the 5.7 branch on percona server github.   We have not in this case, so this is not that understandable to user how to compile.   please note, we have two pages, https://percona-server-docs-5-7-pr-276.onrender.com/installation/git-source-tree.html and also https://percona-server-docs-5-7-pr-276.onrender.com/installation/compile.html we can probably link the first.